AISKU/scripts/listCdnVersions.ps1

```diff
@@ -4,6 +4,8 @@ param (
     [string] $sasToken = $null,                         # The SAS Token to use rather than using or attempting to login
     [string] $logPath = $null,                          # The location where logs should be written
     [switch] $showFiles = $false,                       # Show the individual files with details as well
+    [switch] $inclExt = $false,                         # Include the extensions
```

Comment:
Just enhancing the base list function so you can also list details about the deployed extensions